src/RadixDlt.NetworkGateway.PostgresIntegration/Services/TransactionQuerier.cs

```diff
@@ -186,6 +187,25 @@ public async Task<TransactionPageWithoutTotal> GetTransactionStream(TransactionS
             }
         }
 
+        if (request.SearchCriteria.AffectedGlobalEntities.Any())
+        {
+            userKindFilterImplicitlyApplied = true;
```

Comment:
Are we sure about that? I think non-user TXs can affect various entities.

Reply:
fixed, you're right.